dir.traversal.targets

```diff
@@ -140,7 +142,13 @@
 
       <ProjectsToTest>
        <AdditionalProperties Condition="'%(ProjectsToTest.TestTFMs)'!=''">TestTFMs=%(ProjectsToTest.TestTFMs);%(ProjectsToTest.AdditionalProperties)</AdditionalProperties>
-       <AdditionalProperties>TestTFM=$(FilterToTestTFM);%(ProjectsToTest.AdditionalProperties)</AdditionalProperties>
+      </ProjectsToTest>
+      <ProjectsToTest>
+        <!-- When TestTFMs is not set, and FilterToTestTFM is netcoreapp1.1 -->
+       <AdditionalProperties Condition="('%(ProjectsToTest.TestTFMs)'=='' Or $([System.String]::new('%(Project.TestTFMs)').Contains(';netcoreapp1.0;'))) And '$(FilterToTestTFM)'=='netcoreapp1.1'">TestTFM=netcoreapp1.0;%(ProjectsToTest.AdditionalProperties)</AdditionalProperties>
```

Comment:
Thinking about this a little more I'm not even sure why we are setting the TestTFM property here at all. I can understand defaulting TestTFMs to netcoreapp1.0 if it isn't specified but I don't understand why we would ever set TestTFM directly here. We seem to be trying to handle the logic both here and in dir.props. Can we not combine them in some way to avoid the duplication?